tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.TagsListGenerator.TagListGenerator/AerospikeTags.g.cs

```diff
@@ -2,17 +2,24 @@
 #nullable enable
 
 using Datadog.Trace.Processors;
+using Datadog.Trace.Tagging;
 
 namespace Datadog.Trace.Tagging
 {
     partial class AerospikeTags
     {
-        private static readonly byte[] SpanKindBytes = Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8.GetBytes("span.kind");
-        private static readonly byte[] InstrumentationNameBytes = Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8.GetBytes("component");
-        private static readonly byte[] KeyBytes = Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8.GetBytes("aerospike.key");
-        private static readonly byte[] NamespaceBytes = Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8.GetBytes("aerospike.namespace");
-        private static readonly byte[] SetNameBytes = Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8.GetBytes("aerospike.setname");
-        private static readonly byte[] UserKeyBytes = Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8.GetBytes("aerospike.userkey");
+        // SpanKindBytes = Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8.GetBytes("span.kind");
+        private static readonly byte[] SpanKindBytes = new byte[] { 115, 112, 97, 110, 46, 107, 105, 110, 100 };
```

Comment:
If the goal was to remove the dependency on `Datadog.Trace.Vendors.MessagePack.StringEncoding.UTF8`, we could have just used `System.Text.Encoding.UTF8`. I'm not sure that this actually performs better?

Reply:
The goal is to avoid calling an encoding method when we already can have the static result AOT. I'm replacing here all the call and encoding logic with just a memcpy to create the byte array.

That also opens the possibility to use `ReadOnlySpan<byte>` in the future to remove all allocations for those arrays.